operator/src/main/java/oracle/kubernetes/operator/helpers/JobStepContext.java

```diff
@@ -389,6 +394,26 @@ protected V1Container createPrimaryContainer(TuningParameters tuningParameters)
     return container;
   }
 
+  private String getVolumeName(String resourceName, String type) {
```

Comment:
If these methods do the same thing, why do you need two of them?

Reply:
It was leftover from the previous implementation and I thought having methods with different names might provide better readability. I went ahead and removed it.